django_password_validators/password_character_requirements/password_validation.py

```diff
@@ -1,5 +1,8 @@
 from django.core.exceptions import ValidationError
-from django.utils.translation import ugettext as _, ungettext
+try:
+  from django.utils.translation import gettext as _, ungettext
```

Comment:
@thomasjbradley ungettext was also removed, ngettext should be used.
I believe this was just an easy miss
As I also need this package to upgrade to Django 4.0 I hope this can be merged and released soon

Reply:
Do you think I should alias the function on import: `import ngettext as ungettext` or replace all instances in the code with `ngettext`?

Reply:
Though, if I replace all instances in the code, I’ll have to alias `ungettext` for Django 2 compatibility.

Reply:
Yes, let's keep the django 2 compatibility, you could :

Alias the django 4 import => ngettext as ungettext
This will allow to keep the existing instances in code (validate and get_help_text methods)

OR 

Alias the django 2 & 3 import => ungettext as ngettext
But doing so will need to replace all instances, but also allow to easily drop django 2 support in future releases

(sorry for the english, I'm French)